gbpservice/neutron/config/nsf.ini

```diff
@@ -0,0 +1,13 @@
+[DEFAULT]
+debug=True
+rabbit_password=openstack 
+rabbit_userid=stackrabbit
+rabbit_hosts=<openstack controller ip adress:rabbitPort>  #Example 192.168.2.107:5672
+rabbit_port=5672
+rabbit_host=<openstack controller ip address>
```

Comment:
For configurator we might not need /etc/neutron/neutron.conf -- so this file /etc/nsf/nsf.ini will be useful.. Even inside configurator vm, we decided to use rabbitmq for the communication between rest server and configurator. So, retaining this file for now.